apps/linode-marketplace-drupal/roles/drupal/templates/apache-drupal.conf.j2

```diff
@@ -0,0 +1,32 @@
+<VirtualHost *:80>
+    ServerAdmin webmaster@{{ ansible_fqdn }}
+    DocumentRoot /var/www/drupal/web
+    ServerName {{ ansible_fqdn }}
```

Comment:
we should use `_domain` instead of `ansible_fqdn`

Reply:
`ansible_fqdn` switched to `_domain`